src/IdentityInterface.php

```diff
@@ -53,9 +53,10 @@ public function canResult(string $action, mixed $resource): ResultInterface;
      *
      * @param string $action The action/operation being performed.
      * @param mixed $resource The resource being operated on.
+     * @param mixed $optionalArgs Multiple additional arguments which are passed to the scope
      * @return mixed The modified resource.
      */
-    public function applyScope(string $action, mixed $resource): mixed;
+    public function applyScope(string $action, mixed $resource, mixed ...$optionalArgs): mixed;
```

Comment:
This will be a breaking change as user-land code will have implemented this interface.  We should add this to the documentation/migration guide.

Reply:
Indeed. I was just curious if you guys even like this idea or not 😄 
The first version of the 3.0 migration guide has been added with the major changes I have seen via looking at https://github.com/cakephp/authorization/compare/2.x...3.x-scope-params